pkg/phases/konfigadm.go

```diff
@@ -133,20 +142,34 @@ func baseKonfig(initialKonfigadmFile string) (*konfigadm.Config, error) {
 	return cfg, nil
 }
 
-// addAuditConfig derives the initial admin config for a cluster from its platform
+// addAuditConfig derives the initial audit config for a cluster from its platform
 // config and adds it to its konfigadm files
 func addAuditConfig(platform *platform.Platform, cfg *konfigadm.Config) error {
-	if platform.Kubernetes.AuditConfig != nil && platform.Kubernetes.AuditConfig.PolicyFile != "" {
+	if pf := platform.Kubernetes.AuditConfig.PolicyFile; pf != "" {
```

Comment:
I switched the `AuditConfig` section to not be given by value not reference, simplifies the checking everywhere.

https://github.com/flanksource/platform-cli/blob/4018e1b965f5a284cbffa38cf69611bd51c1b350/pkg/types/types.go#L303